drivers/ethernet/eth_nxp_enet_qos/eth_nxp_enet_qos_mac.c

```diff
@@ -254,22 +288,46 @@ static void eth_nxp_enet_qos_mac_isr(const struct device *dev)
 	enet_qos_t *base = config->base;
 
 	/* cleared on read */
-	volatile uint32_t mac_interrupts = base->MAC_INTERRUPT_STATUS;
-	volatile uint32_t mac_rx_tx_status = base->MAC_RX_TX_STATUS;
-	volatile uint32_t dma_interrupts = base->DMA_INTERRUPT_STATUS;
-	volatile uint32_t dma_ch0_interrupts = base->DMA_CH[0].DMA_CHX_STAT;
+	uint32_t mac_interrupts = base->MAC_INTERRUPT_STATUS;
+	uint32_t mac_rx_tx_status = base->MAC_RX_TX_STATUS;
+	uint32_t dma_interrupts = base->DMA_INTERRUPT_STATUS;
+	uint32_t dma_ch0_interrupts = base->DMA_CH[0].DMA_CHX_STAT;
 
-	mac_interrupts; mac_rx_tx_status;
+	/* clear pending bits except RBU
```

Comment:
Now RBU is also cleared here, so maybe the first part of the comment is a little confusing.